test/tests/redfish10/test_redfish10_system_reset_base.py

```diff
@@ -152,8 +145,33 @@ def workflow_tasklist_status_poller(tasklist, tasktype, timeout=180):
 
     return task_errorlist
 
+
+def get_rackhd_nodetype(nodeid):
```

Comment:
There is a limited scope to this PR. We can't fix everything here. This PR needs to be merged before API 1.1 is deprecated, which is imminent. Lets note all of the shortcomings in these files and build stories to address them.